frontend/src/metabase/ui/components/feedback/Skeleton/Skeleton.styled.tsx

```diff
@@ -16,10 +16,10 @@ export const getSkeletonOverrides = (): MantineThemeOverride["components"] => {
         return {
           // We replace Mantine's pulsing animation with a shimmer animation
           root: {
-            backgroundColor: "rgba(0, 0, 0, .03)",
+            backgroundColor: "rgba(245,248,250,1.0)",
```

Comment:
do we have a css var for this??

Reply:
I don't think we do :/

Reply:
i shall call upon the color crew

Reply:
It's basically a slightly darker bg-light. Maybe we can use color-mix to get that?

Reply:
looks like if we mix `bg-light` and `bg-medium` at 35%, we get the resulting color haha

Reply:
I think we want to avoid merging and use actual colors for this. let's merge this as-is and follow up later following @kdoh 's suggestions [in this thread](https://metaboat.slack.com/archives/C057WD5L0JG/p1727896989103999)